src/TestRunner.js

```diff
@@ -84,23 +91,49 @@ export default class TestRunner {
   // 3. Re-renders the app
   // 4. Runs the test
   async runTest(scope, test) {
+    const startTime = new Date();
+
     await this.component.clearAsync();
+
     if (scope.beforeEach) { await scope.beforeEach.call(scope) };
+    
     this.component.reRender();
 
     // Run the test, console logging the result.
-    let { description, f } = test;
+    let { title, label, description, f } = test;
+
+    if (!this.results.some(suite => suite.name == title)) {
+      this.results.push({
```

Comment:
Could you create a reference to this object and then use it below instead of re-finding it?

Reply:
Approach changed - no rolling up of cases into suites.